### PR TITLE
Feat/v1

### PR DESCRIPTION
**Pull request by nayan-mistry**
V1 of Twilio Sendgrid send email action

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,16 +1,97 @@
...
+      text: emailBodyText,
+      html: emailBodyHtml
+    }
+
+    if (hasAttachments) {
+      //currently we are supporting only single attachment      
+      const attachmentFileName = file.parseFileName(attachmentsPath)
+      const attachmentContent = await file.getEncodedFileContents(attachmentsPath)
+      
+      core.debug(`Attachment Content - ${attachmentContent}`)
```
Comment (EricBartusch) [inline src/main.ts:59]: Do we want to leave these in?

File: __tests__/main.test.ts [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 
 test('throws invalid number', async () => {
-  const input = parseInt('foo', 10)
-  await expect(wait(input)).rejects.toThrow('milliseconds not a number')
+  const input = 10
+  await expect(input).toBeGreaterThan(1)
 })
 
-test('wait 500 ms', async () => {
+/*test('wait 500 ms', async () => {
```
Comment (EricBartusch) [inline __tests__/main.test.ts:10]: Can we remove the commented out lines.

File: src/main.ts (hunk 1 above)
Comment (nayan-mistry) [reply src/main.ts:59]: good catch! i remove this.

File: src/main.ts [hunk 3]
```diff
@@ -1,16 +1,96 @@
...
+      text: emailBodyText,
+      html: emailBodyHtml
+    }
+
+    if (hasAttachments) {
+      //currently we are supporting only single attachment      
+      const attachmentFileName = file.parseFileName(attachmentsPath)
+      const attachmentContent = await file.getEncodedFileContents(attachmentsPath)
+            
+      core.debug(`Attachment Filename - ${attachmentFileName}`)
```
Comment (EricBartusch) [inline src/main.ts:59]: Let's remove the other two debug statements too. Unless you wanted them in for some reason.

**Review by EricBartusch — APPROVED** [review]

**Pull request merged**